src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,55 @@
+.stop-watch {
+  position: relative;
+  display: flex;
+  @include square($stop-watch-width-and-height);
+  background-color: rgba(72, 33, 72, 0.82);
+  border: 3px solid transparent;
+  border-radius: 100%;
+
+  &__arrow {
+    position: absolute;
+    bottom: $arrow-bottom;
+    left: 50%;
+    width: 5px;
+    border: 1px solid transparent;
+    border-radius: 30px;
+    background-color: $color;
+    animation-name: arrow;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    transform-origin: bottom;
+
+    &--minutes {
+      height: 120px;
+      animation-duration: 3600s;
+      animation-delay: 60s;
```

Comment:
do you really need it here? your minute arrow start after 60s delay. 61minute in one hour? fix it, please
```suggestion
     
```